homeassistant/components/config/__init__.py

```diff
@@ -112,10 +114,10 @@ async def get(self, request, config_key):
 
         return self.json(value)
 
-    async def post(self, request, config_key):
+    async def post(self, request: web.Request, config_key: str) -> web.Response:
         """Validate config and return results."""
         try:
-            data = await request.json()
+            data = await request.json(loads=json_loads)
```

Comment:
Isn’t it possible to set a Request base class on the web server? Wouldn’t that allow us to set this globally

Reply:
Looks like its possible on the `client` side only

```
        request_class: Type[ClientRequest] = ClientRequest,
        response_class: Type[ClientResponse] = ClientResponse,
```

Reply:
I think this might work https://github.com/home-assistant/core/pull/90214

Reply:
I was able to make it work. I removed everything except for emulated_hue from this PR since it has its own app